stacks/apps/exchange.yml

```diff
@@ -1537,7 +1464,7 @@ Resources:
     Properties:
       DelaySeconds: 0
       MessageRetentionPeriod: 345600 # 4 days
-      QueueName: !Sub ${Constants.SqsQueuePrefix}_exchange_porter_callback
+      QueueName: !Sub ${Constants.SqsQueuePrefix}porter_callback
```

Comment:
this resulted in `dc51b3fd_prod_exchange__exchange_porter_callback`, which is not awesome, duplicate `_exchange`

Reply:
This does go with a change to the exchange app's config/messaging.rb as well.